prow/cmd/clonerefs/Dockerfile

```diff
@@ -15,5 +15,6 @@
 FROM gcr.io/k8s-prow/git:0.1
 LABEL maintainer="skuznets@redhat.com"
 
+RUN git config --global user.name ci-robot && git config --global user.email ci-robot@k8s.io
```

Comment:
can we do this at runtime instead? it seems like this should be configurable for different prow deployments without building a new image

Reply:
Sure, I can add flags on `clonerefs` to expose these two instead